concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md

```diff
@@ -76,6 +120,63 @@ To process a new **ColorBox** value, you need to handle the value change event.
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxColorBox
+            :onValueChanged="colorBox_valueChanged"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxColorBox from 'devextreme-vue/color-box';
+
+    export default {
+        components: {
+            DxColorBox
+        },
+        data() {
+            return {
+                colorBox_valueChanged: function (e) {
+                    let previousValue = e.previousValue;
+                    let newValue = e.value;
+                    // Event handling commands go here
+                }
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import ColorBox from 'devextreme-react/color-box';
+
+    class App extends React.Component {
+        colorBox_valueChanged(e) {
```

Comment:
I suggest renaming the function in React and Angular approaches based on my suggestion above.

Reply:
fixed